spec/player_input_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative "../lib/player_input"
+
+RSpec.describe PlayerInput do
+  describe "#get_coordinates" do
+    it "returns the user's input" do
+      input = double
+      mock_view = double
+      player_turn = PlayerInput.new(input: input, view: mock_view)
+      allow(mock_view).to receive(:render_coordinates_prompt)
```

Comment:
Since you aren't making assertions on the mock_view. It would be easier for me to ignore this setup if it was built into the double: `mock_view = double(render_coordinates_prompt: nil, render_coordinates_error: nil)`. Then I just kind of ignore it and assume that its role is just as an accepting collaborator.